boards/arm/wio_terminal/doc/index.rst

```diff
@@ -0,0 +1,219 @@
+| Flash     | on-chip    | Can be used with LittleFS to store files |
++-----------+------------+------------------------------------------+
+| SYSTICK   | on-chip    | Systick                                  |
++-----------+------------+------------------------------------------+
+| WDT       | on-chip    | Watchdog                                 |
++-----------+------------+------------------------------------------+
+| GPIO      | on-chip    | I/O ports                                |
++-----------+------------+------------------------------------------+
+| USART     | on-chip    | Serial port                              |
++-----------+------------+------------------------------------------+
+| I2C       | on-chip    | Inter-Integrated Circuit                 |
++-----------+------------+------------------------------------------+
+| SPI       | on-chip    | Serial Peripheral Interface port         |
++-----------+------------+------------------------------------------+
+| TRNG      | on-chip    | True Random Number Generator             |
++-----------+------------+------------------------------------------+
+| HWINFO    | on-chip    | Unique 128 bit serial number             |
++-----------+------------+------------------------------------------+
+| RTC       | on-chip    | Real-Time Counter                        |
++-----------+------------+------------------------------------------+
+| USB       | on-chip    | USB device                               |
++-----------+------------+------------------------------------------+
+| PWM       | on-chip    | PWM                                      |
++-----------+------------+------------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig file
+:zephyr_file:`boards/arm/wio_terminal/wio_terminal_defconfig`.
+
+Zephyr can use the default Cortex-M SYSTICK timer or the SAM0 specific RTC.
+To use the RTC, set :code:`CONFIG_CORTEX_M_SYSTICK=n` and set
+:code:`CONFIG_SYS_CLOCK_TICKS_PER_SEC` to no more than 32 kHZ divided by 7,
+i.e. no more than 4500.
+
+Connections and IOs
+===================
+
+The `Wio Terminal Getting started guide`_ has detailed
+information about the board including `pinouts`_ and the `schematic`_.
+
+System Clock
+============
+
+The SAMD51 MCU is configured to use the 32.768 kHz internal oscillator
+with the on-chip PLL generating the 120 MHz system clock.
+
+Serial Port
+===========
+
+The SAMD51 MCU has 8 SERCOM based USARTs. On the Wio Terminal, no SERCOM
+is available for the Zephyr console but it is possible to use SERCOM2
+available on Raspberry Pi compatible header pins 8 (TX) and 10 (RX).
+
+To add Zephyr console over Raspberry Pi connector, add the following
+configuration and overlay in your project:
+
+```
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+```
+
+```
+/ {
+    chosen {
+        zephyr,console = &raspberrypi_serial;
```

Comment:
Because there is no real console port on the Wio Terminal. This is a serial port available on the Raspberry Pi connector. Not even very easy to connect an FTDI or something similar without a bit of wiring.
So best choice is no console by default. I have added here this comment because many people will look for a console. This is a solution, even if the USB console is very nice too.